<launch>
    <node name="sign_detector" pkg="rr_iarrc" type="sign_detector" output="screen" required="true">
        <!-- Sign Detector Params -->
        <param name="front_image_subscription" type="string" value="/camera_center/image_color_rect"/>
        <param name="sign_string_publisher" type="string" value="/turn_detected"/>

        <!--Crop Area for detecting signs (-1 keeps whole screen)-->
        <param name="roi_x" type="int" value="600" />
        <param name="roi_y" type="int" value="300" />
        <param name="roi_width" type="int" value="680" />
        <param name="roi_height" type="int" value="350" />

        <param name="sign_file_package_name" type="string" value="rr_iarrc"/>
        <param name="sign_file_path_from_package" type="string" value="/src/sign_detector/sign_forward.jpg"/>

        <param name="template_confidence_threshold" type="double" value="0.8" />
        <param name="scalars_start" type="double" value="0.3" />
        <param name="scalars_end" type="double" value="1.2" />
        <param name="scalars_num" type="int" value="5" />

    </node>
</launch>
